### PR TITLE
ansible/helm: add makefile helper

### PR DESCRIPTION
**Pull request by camilamacedo86**
Signed-off-by: Camila Macedo <cmacedo@redhat.com>

**Description of the change:**
add makefile helper for ansible/helm

**Motivation for the change:**
provide for ansible/helm the same nice features that we have for Go
Closes: https://github.com/operator-framework/operator-sdk/issues/4628

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by jberkhahn** [comment]
seems to work fine. I worry about this just being text in a template somewhere, though - one more thing we have to remember to update if anything changes.
/lgtm

File: changelog/fragments/ansible-helm-makefile-help.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (helm/v1)(ansible/v1) - Add Makefile helper
```
Comment (estroz) [inline changelog/fragments/ansible-helm-makefile-help.yaml:5]:
```suggestion
      (helm/v1, ansible/v1) Added Makefile `help` target
```

File: changelog/fragments/ansible-helm-makefile-help.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (helm/v1,ansible/v1) - **(Optional)** Add Makefile helper
+      body: >
+        Ansible/Helm projects scaffolded by the tool now provides a Makefile
+        helper. To address these changes you can copy and paste the Makefile from
+        its samples which are in the testdata. Check the latest version of the
+        Helm Makefile [here](https://github.com/operator-framework/operator-sdk/blob/v1.6.0/testdata/helm/memcached-operator/Makefile) and check it for Ansible [here](https://github.com/operator-framework/operator-sdk/blob/v1.6.0/testdata/ansible/memcached-operator/Makefile).
```
Comment (estroz) [inline changelog/fragments/ansible-helm-makefile-help.yaml:36]:
```suggestion
        Ansible/Helm projects now provide a Makefile `help` target, similar to a `--help` flag.
        You can copy and paste this target from the relevant sample's Makefile
        ([helm]((https://github.com/operator-framework/operator-sdk/blob/v1.5.0/testdata/helm/memcached-operator/Makefile), [ansible]((https://github.com/operator-framework/operator-sdk/blob/v1.5.0/testdata/ansible/memcached-operator/Makefile)).
```

File: changelog/fragments/ansible-helm-makefile-help.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (helm/v1,ansible/v1) - **(Optional)** Add Makefile helper
```
Comment (estroz) [inline changelog/fragments/ansible-helm-makefile-help.yaml:31]:
```suggestion
      header: (helm/v1, ansible/v1) Add `help` target to Makefile.
```

**Review by estroz — CHANGES_REQUESTED** [review]

File: internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go [hunk 4]
```diff
@@ -70,40 +70,52 @@ IMG ?= {{ .Image }}
...
-	docker push ${IMG}
-
 OS := $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH := $(shell uname -m | sed 's/x86_64/amd64/')
 
-# Download kustomize locally if necessary, preferring the $(pwd)/bin path over global if both exist.
-.PHONY: kustomize
+.PHONY: kustomize ## Download kustomize locally if necessary.
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize:
```
Comment (estroz) [inline internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:120]:
```suggestion
.PHONY: kustomize
KUSTOMIZE = $(shell pwd)/bin/kustomize
kustomize: ## Download kustomize locally if necessary.
```

File: internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go [hunk 5]
```diff
@@ -119,8 +131,7 @@ KUSTOMIZE = $(shell which kustomize)
 endif
 endif
 
-# Download ansible-operator locally if necessary, preferring the $(pwd)/bin path over global if both exist.
-.PHONY: ansible-operator
+.PHONY: ansible-operator ## Download ansible-operator locally if necessary, preferring the $(pwd)/bin path over global if both exist.
 ANSIBLE_OPERATOR = $(shell pwd)/bin/ansible-operator
 ansible-operator:
```
Comment (estroz) [inline internal/plugins/ansible/v1/scaffolds/internal/templates/makefile.go:136]:
```suggestion
.PHONY: ansible-operator
ANSIBLE_OPERATOR = $(shell pwd)/bin/ansible-operator
ansible-operator: ## Download ansible-operator locally if necessary, preferring the $(pwd)/bin path over global if both exist.
```

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**